server/superdesk/errors.py

```diff
@@ -312,3 +312,33 @@ def ftpUnknownParserError(cls, exception, provider, filename):
             logger.exception("Provider: {} - File: {} unknown file format. "
                              "Parser couldn't be found.".format(provider.get('name', 'Unknown provider'), filename))
         return IngestFtpError(5001, exception, provider)
+
+
+class IngestRssFeedError(SuperdeskIngestError):
```

Comment:
not sure we need rss specific error class, eg. auth error would make sense also for reuters where you have to authenticate. also connection timeout would work for http apis, not found too.